osu.Game.Rulesets.Catch/UI/SkinnableCatcher.cs

```diff
@@ -26,8 +26,9 @@ public SkinnableCatcher()
             : base(new CatchSkinComponentLookup(CatchSkinComponents.Catcher), _ => new DefaultCatcher())
         {
             Anchor = Anchor.TopCentre;
-            // Sets the origin roughly to the centre of the catcher's plate to allow for correct scaling.
-            OriginPosition = new Vector2(0.5f, 0.06f) * Catcher.BASE_SIZE;
+            // Sets the origin to the top of the catcher's plate to allow for correct scaling.
+            OriginPosition = new Vector2(0.5f, 0.0525f) * Catcher.BASE_SIZE;
```

Comment:
The derivation of this `0.0525f` magic constant should be spelled out in comments. I refuse to spend time on reconstructing the math. The OP only links random xmldoc:

> see [this xmldoc](https://github.com/ppy/osu/blob/649c447265c30a2f214442ec5b633d12e13821ff/osu.Game.Rulesets.Catch/UI/Catcher.cs#L34) and [stable's definition of `catcherWidth`](https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameModes/Play/Rulesets/Fruits/RulesetFruits.cs#L235) for more information.

I don't know how you get from any of these above to `0.0525f`.

If you have this stuff calculated and cross-referenced already, the _least_ you can do is to provide it for double-checking.

Reply:
This part is ballparked and can be reverted I believe, the comment already states it's a close value, but I guess that's not quite clear.

Reply:
@frenzibyte  Are you going to redo visual comparisons after attempting the revert?

Reply:
Sure, yeah.